woocommerce/woocommerce-android
# [Custom Amounts M4] Change custom amounts dialog to full screen

Closes: #10393 


### Description

This PR transforms the custom amount dialog into a full-screen fragment. We made this change due to the considerable amount of information on the custom amount screen, which proved to be overwhelming for a dialog format, especially in landscape mode. The transition to a full-screen fragment enhances the user interface and ensures better usability across different orientations.

### Testing instructions

1. Navigate to the order creation screen
2. Click on "Add Custom Amount" row
3. Ensure the custom amount is a full screen fragment
4. Ensure the custom amount UI behaves properly in the landscape mode

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/1331230/9804fe2d-1968-407d-b267-d15c70457b35
